main/main.cpp

```diff
@@ -18,7 +18,6 @@
 //=============================================================================
 
 #include "config.h"
-#include "mscore/musescore.h"
 #include "modulessetup.h"
 
 #ifdef BUILD_UI_MU4
```

Comment:
You missed this

Reply:
I plan to do it little by little. One of the next steps will be to remove this option. Thanks

Reply:
"Rough strokes" are now done :)